users/service.go

```diff
@@ -255,14 +335,23 @@ func (svc service) UpdateEmail(ctx context.Context, session authn.Session, userI
 			return User{}, err
 		}
 	}
+	oldUsr, err := svc.users.RetrieveByID(ctx, userID)
+	if err != nil {
+		return User{}, errors.Wrap(svcerr.ErrUpdateEntity, err)
+	}
+	if oldUsr.Email == email {
+		return User{}, fmt.Errorf("current email is same as update requested email")
+	}
 
-	updatedAt := time.Now().UTC()
-	usr := UserReq{
-		Email:     &email,
-		UpdatedAt: &updatedAt,
-		UpdatedBy: &session.UserID,
+	usr := User{
+		ID:         userID,
+		Email:      email,
+		UpdatedAt:  time.Now().UTC(),
+		UpdatedBy:  session.UserID,
+		VerifiedAt: time.Time{},
 	}
-	user, err := svc.users.Update(ctx, userID, usr)
+
+	user, err := svc.users.UpdateEmail(ctx, usr)
```

Comment:
Should we send a verification email?

Reply:
No we can send email on demand of User request    

For we can have verification button in UI , After successfull change of email
Or   
For better user experince, This can be done in UI, After successfull change of email , UI can send send verification request to backend.